src/controllers/dynakube/version/image_hash.go

```diff
@@ -69,11 +101,10 @@ func GetImageVersion( //nolint:revive // argument-limit
 	if dynakube.Spec.TrustedCAs != "" {
 		transport, err = addCertificates(transport, dynakube, apiReader)
 		if err != nil {
-			return registry.ImageVersion{}, fmt.Errorf("addCertificates(): %w", err)
+			return nil, nil, fmt.Errorf("addCertificates(): %w", err)
```

Comment:
use `errors.WithMessagef` instead of `fmt.Errorf`

Reply:
@0sewa0 can we add a linter rule? I saw similar strict rules in another projects.

Reply:
Would it make sense to use errors.Wrap also?
`errors.Wrap(err, "failed adding trusted CAs to transport")`
Or is it better practice to use `errors.WithMessagef`

Reply:
@0sewa0 I'll create a small research spike to add a linter rule or document it in a coding-style guide.

Reply:
Fixed!